### ISSUE TITLE
Document end-to-end testing

### ISSUE THREAD

**Issue by yoomlam** [department-of-veterans-affairs/abd-vro#644]
Document end-to-end testing (contact Kalai, Amy, and Emily)
  - [VRO Test Cases](https://github.com/department-of-veterans-affairs/abd-vro/wiki/VRO-Test-Cases)
  - [Slack info](https://dsva.slack.com/archives/C01Q7979Z7D/p1667489767662659)

**Comment by amylai-nava** [comment]
**Additional docs to review**:
- [Test cases](https://dvagov.sharepoint.com/:f:/r/sites/vaabdvro/Shared%20Documents/VRO%20-%20Virtual%20Regional%20Office/End-to-end%20Testing?csf=1&web=1&e=xxt4wO)
- ABD test users list (access restricted)
- [Requesting additional or modifications to LH test users](https://dvagov.sharepoint.com/:b:/r/sites/vaabdvro/Shared%20Documents/RRD%201st%20gen/[Lighthouse_Test_Users.pdf](https://dvagov.sharepoint.com/:b:/r/sites/vaabdvro/Shared%20Documents/RRD%201st%20gen/Lighthouse_Test_Users.pdf?csf=1&web=1&e=bTXnyx)?csf=1&web=1&e=bTXnyx)
- [Test user setup in VBMS](https://dvagov.sharepoint.com/:b:/r/sites/vaabdvro/Shared%20Documents/RRD%201st%20gen/[Lighthouse_Test_Users.pdf](https://dvagov.sharepoint.com/:b:/r/sites/vaabdvro/Shared%20Documents/RRD%201st%20gen/Lighthouse_Test_Users.pdf?csf=1&web=1&e=bTXnyx)?csf=1&web=1&e=bTXnyx) (needs updating; recommend getting access to cancel EPs)
- [Process for adding test users to VBMS if more are needed](https://dsva.slack.com/archives/C01BVF2A3V0/p1662068255268139)
- VA.gov test user dashboard [questions ](https://dsva.slack.com/archives/CBU0KDSB1/p1662067938628599)for the va.gov platform team
- [More about test users](https://dsva.slack.com/archives/C01Q7979Z7D/p1657735400964759?thread_ts=1657734603.456949&cid=C01Q7979Z7D)
- [BGS contact](https://dsva.slack.com/archives/C01Q7979Z7D/p1664983668229519) for clearing EPs if we are unable to do so in VBMS (see VA email thread)
- [Testing environments](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Development-environments#vro-v1-testing)
- [This ](https://dsva.slack.com/archives/C01Q7979Z7D/p1663351583382399)might be relevant?

**Comment by yoomlam** [comment]
Include details like [ICNs](https://dsva.slack.com/archives/C01Q7979Z7D/p1668118747091689?thread_ts=1668114587.350889&cid=C01Q7979Z7D) and submission data to help diagnose problems.

**Comment by amylai-nava** [comment]
@dianagriffin cc: @yoomlam Here's the [document](https://docs.google.com/document/d/1790yCXE1v8EXGyVS1RpzkIlJt-aeOikbUMu2a5Q2HNY/edit?usp=sharing) I started last week. I will likely need to pick this back up after Thanksgiving, but posting it here to share progress. Feel free to add to it.

**Comment by dianagriffin** [comment]
@amylai-nava I think we should break this one down into more specific tickets, since we need to prioritize documenting what's needed to support iMVP testing but the overall documentation of everything that will be helpful for future testing is a bigger task that isn't as high priority. wdyt?

**Comment by amylai-nava** [comment]
@dianagriffin Makes sense to me! What I've worked on is for the latter and not for iMVP testing. Do you need any support from me for iMVP testing documentation?

**Comment by amylai-nava** [comment]
@dianagriffin FYI, I moved this out of sprint L. Also adding a reminder for us to chat about this next week and see if it should be prioritized for sprint M.
